README.md

```diff
@@ -140,6 +140,8 @@ You can do this in the Vercel Project dashboard under _"Settings > Environment V
 
 > Hint: You can mass copy all the config variables from your `packages/nextjs/.env.local` config files and paste them into the Vercel form.
 
+> Note: If you run `yarn vercel` from the CLI, after setting all environment variables in Vercel, you will need to perform a second deployment to refresh them, by running `yarn vercel` again.
```

Comment:
Maybe we could generalize a bit more because I think if you connect the repo through github you need to trigger a redeploy again too : 

![Screenshot 2023-10-09 at 9 17 31 AM](https://github.com/BuidlGuidl/PWA-burner-wallet/assets/80153681/e1e4e68f-8a43-4a22-81b1-a6f544f9fd22)

^ here the repo was connected through GitHub and I tried adding env's now but it didn't re-trigger deploy

Reply:
Agree! I barely use `yarn vercel`

Maybe we can have something more generic, mentioning that you need to redeploy after setting the env vars.

Reply:
Don't know why I thought that refresh would only affect when deploying to Vercel from console 🤷‍♂️🤯

I've just pushed the changes on the explanation, LMK if you would change something else ♥